addons/kotsadm/alpha/operator.yaml

```diff
@@ -79,3 +79,10 @@ spec:
           valueFrom:
             fieldRef:
               fieldPath: metadata.namespace
+      resources:
+        limits:
```

Comment:
these limits seem aggressive

Reply:
It successfully applied over 200 new deployments at the same time in my testing

Reply:
And deleted+recreated the same: 
![image](https://user-images.githubusercontent.com/2318911/90548241-a634b600-e15a-11ea-8611-83b49b97b84a.png)

Test app yam was ~10,000 lines of the following:
```
apiVersion: apps/v1
kind: Deployment
metadata:
  name: example-nginx-many-repl{{ RandomString 8 "[abcdefghijklmnnopqrstuvwxyz]" }}
  labels:
    app: example
    component: nginx-many
spec:
  replicas: 1
  selector:
    matchLabels:
      app: example
      component: nginx-many
  template:
    metadata:
      labels:
        app: example
        component: nginx-many
    spec:
      containers:
        - name: nginx-2
          image: docker.io/library/nginx:1.17.7
          env:
          - name: CONFIG_STRING
            value: 'repl{{ ConfigOption "a_test_string" }}'
          envFrom:
          - configMapRef:
              name: example-config
          ports:
          - containerPort: 3001
            name: comply-ui
            protocol: TCP
          resources:
            requests:
              memory: '16Mi'
              cpu: '5m'
            limits:
              memory: '32Mi'
              cpu: '50m'
          volumeMounts:
          - name: config-volume
            mountPath: "/etc/nginx"
      volumes:
      - name: config-volume
        configMap:
          name: nginx2-config
---
```